common/src/main/java/org/apache/druid/metadata/MetadataStorageConnectorConfig.java

```diff
@@ -44,6 +53,9 @@
   @JsonProperty("password")
   private PasswordProvider passwordProvider;
 
+  @JsonProperty("dbcpPropertiesFile")
+  private String dbcpPropertiesFile;
```

Comment:
this requires users to deploy a new configuration file... how about changing above to something like...

```
@JsonProperty("dbcp")
 private Properties dbcpProperties;

public Properties getDbcpProperties()
{
  return dbcpProperties == null ? new Properties() : dbcpProperties;
}
```

then user can just put following kind of things in druid runtime.properties file...

```
druid.metadata.storage.connector.dbcp.maxConnLifetimeMillis=1200000
druid.metadata.storage.connector.dbcp.defaultQueryTimeout=30000
....
```

I think that should work..

Reply:
sure, let me try it